bin/pycbc_multi_inspiral

```diff
@@ -382,11 +376,21 @@ with ctx:
     }
     del time_delay_idx_zerolag
 
-    # Matched filter each ifo. Do not cluster here for a coherent search.
-    # Clustering happens at the end of the template loop.
+    logging.info("Setting up MatchedFilterControl at each IFO")
+    # Prototype container for the output of MatchedFilterControl and
+    # waveform.FilterBank (see below).
+    # Use tlen of the first IFO as it is the same across IFOs.
+    template_mem = zeros(tlen, dtype=complex64)
+
+    # We are currently using the same matched-filter parameters for all
+    # IFOs. Therefore only one MatchedFilterControl setup is needed.
+    # This can be change if needed.
```

Comment:
I actually see one MatchedFilterControl for each detector, so is this comment still appropriate?

Reply:
Let me propose a different text.  What I meant is that (`segments[ifo]` aside) we call `MatchedFilterControl` the exact same way on all available detectors, whereas we would want a single IFO SNR threshold just for the two most sensitive ones.  This would require a non-zero `args.snr_threshold` for 2 detectors and a vanishing one for all other the others.

Reply:
```
    # We are currently using the same matched-filter parameters for all
    # IFOs. Therefore we loop over one MatchedFilterControl setup.
    # This can be change if needed.
```

Reply:
Ah I see, then maybe rephrase "only one MatchedFilterControl setup is needed" as "all MatchedFilterControl instances are initialized in the same way"?

Reply:
```
    # We are currently using the same matched-filter parameters for all
    # IFOs. Therefore only all MatchedFilterControl instances are initialized
    #  in the same way. This can be change if needed.
```